Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+    public Rigidbody2D body;
+    public SpriteRenderer sprite;
+    public Animator animator;
+    public float speed = 8.0f;
+
+    Vector3 moveDirection = Vector3.zero;
+    string lastAnimation = "";
+
+
+    void Start() 
+    {
+        SetAnimation("idle_down");
+    }
+
+    void Update()
+    {
+        UpdateAnimator();
+    }
+
+    void FixedUpdate()
+    {
+        moveDirection = new Vector3(
+            Input.GetAxisRaw("Horizontal"), 
+            Input.GetAxisRaw("Vertical"), 
```

Comment:
GetAxisRaw will not smooth the keyboard inputs, so youll get -1, 0 or 1. I think its more appropriate to use Get Axis

Reply:
That one is intentional, with input smoothing, it feels like driving a car instead of someone. Using raw inputs allows for a sharper/nervous gameplay

Reply:
So no drag? Im ok to trust on this though